com.valvesoftware.Steam.yml

```diff
@@ -117,6 +117,12 @@ add-extensions:
     no-autodownload: true
     autodelete: false
 
+  com.valvesoftware.Steam.CompatibilityTool.Proton:
+    directory: share/steam/compatibilitytools.d/Proton
+    version: beta
+    versions: beta;test
```

Comment:
Is the `test` intentional?

Reply:
Yes, it's same as for the multi-extension point.